app/controllers/api/v1/activities_controller.rb

```diff
@@ -6,6 +6,9 @@ def latest
 
   def just_updated
     versions = Version.just_updated(50)
+    if params[:since].present?
```

Comment:
@sonalkr132 Documentation is something that definitly needs to be added :-) At this point I expect a timestamp. So maybe a simple `to_i` would suffice?